internal/controller/postgrescluster/cluster_test.go

```diff
@@ -47,6 +47,8 @@ import (
 	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
 )
 
+func Int32(v int32) *int32 { return &v }
```

Comment:
I think we can use `initialize.Int32()` here, maybe?

https://github.com/CrunchyData/postgres-operator/blob/master/internal/initialize/primitives.go#L29

Reply:
Ah, ok, I removed and replaced, thanks for the reference